specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/AppServiceEnvironments.json

```diff
@@ -747,6 +747,68 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/hostingEnvironments/{name}/migrate": {
+      "post": {
+        "tags": [
+          "AppServiceEnvironments"
+        ],
+        "summary": "Migrate an App Service Environment between ASE versions.",
+        "operationId": "AppServiceEnvironments_Migrate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the App Service Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "phase",
+            "in": "query",
+            "description": "Specify <code>Validation</code> to validate, <code>PreMigration</code> to generate new IP addresses and <code>FullMigration</code> to migrate the ASE.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "migrationOptionsEnvelope",
+            "in": "body",
+            "description": "Additional options and configurations that will be applied to the App Service Environment during migration.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AseMigrationOptions"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Operation is in progress."
+          },
```

Comment:
Why does it not have a response schema? Without a response, how does use get e.g. "provisioningState" property?

And is it LRO?